### DISCUSSION TITLE
Unable to send logs from fluent-bit to influxdb (cpu, mem, etc. work perfectly)

### DISCUSSION THREAD

**Discussion by FelixVillanueva** [fluent/fluent-bit#9320] [Q&A]
Any try to send logs from a machine running debian with fluentbit v3.1.6 to a server running influxdb provides:
...
[ warn] [output:influxdb:influxdb.0] skip send record, since no record available or all fields are tagged in record
...

I can sucessfully send memory statistics, cpu, etc.. but systemd, specific log inputs, etc. always give me the error above

Minimun fluentbit.conf :

[INPUT]
    Name   systemd
    Tag    k3s.logs
    Systemd_Filter _SYSTEMD_UNIT=k3s.service
    Read_From_Tail on

[FILTER]
    Name record_modifier
    Match *
    Record hostname ${HOSTNAME}

[OUTPUT]
    Name        influxdb
    Match       *
    Host        172.24.27.38
    Port        8086
    Database    TALENT-BELIEF
    Org         inwire
    http_token  xxxxx
    Bucket      TALENT-BELIEF
    Auto_Tags   true
    Sequence_Tag _seq

[OUTPUT]
    Name     stdout
    Match    *

I tried all type of different INPUT options, rename records, add records, etc. without success

**Comment by patrick-stephens** [comment]
Influxdb is a time series database for metrics, not logs. You need to convert logs to metrics in some way, e.g. `log_to_metrics` filter: https://docs.fluentbit.io/manual/pipeline/filters/log_to_metrics
